src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/RECAPTCHAManager.java

```diff
@@ -0,0 +1,83 @@
+package uk.ac.cam.cl.dtg.segue.api.managers;
+
+import com.google.inject.Inject;
+import org.apache.commons.lang3.Validate;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.ac.cam.cl.dtg.util.PropertiesLoader;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import static uk.ac.cam.cl.dtg.segue.api.Constants.GOOGLE_RECAPTCHA_SECRET;
+
+public class RECAPTCHAManager {
+    private final PropertiesLoader properties;
+    private static final Logger log = LoggerFactory.getLogger(RECAPTCHAManager.class);
+
+    @Inject
+    public RECAPTCHAManager(final PropertiesLoader properties) {
+        Validate.notNull(properties.getProperty(GOOGLE_RECAPTCHA_SECRET));
+        this.properties = properties;
+    }
+
+    public String isCaptchaValid(final String response) {
+        if (response == null || response.isEmpty()) {
+            return "Missing reCAPTCHA response token.";
+        }
+
+        if (verifyRecaptcha(response)) {
+            return "reCAPTCHA verification successful.";
+        } else {
+            return "reCAPTCHA verification failed.";
+        }
+    }
+
+    protected JSONObject performHttpRequest(final String url, final String params) throws Exception {
```

Comment:
Try to use more specific exceptions in your throws rather than the generic Exception.